From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Sat, 2 Jan 2021 20:00:00 +0100
Subject: [PATCH 1/1] Add llvm-mingw armv7 dlltool target

Upstream-Status: Inappropriate [enable feature]
armv7-pc-windows-gnullvm is not an officially supported Rust target.

diff --git a/src/build.rs b/src/build.rs
index 1111111..2222222 100644
--- a/src/build.rs
+++ b/src/build.rs
@@ -221,6 +221,7 @@ fn build_implib_file(
                 "x86_64" => "i386:x86-64",
                 "x86" => "i386",
                 "aarch64" => "arm64",
+                "armv7" => "arm",
                 _ => unimplemented!("Windows support for {} is not implemented yet.", arch),
             };
 
